docker/wiremock/stubs/mappings/retrievePatientInvalidNhsNumber.json

```
@@ -2,19 +2,19 @@
   "priority": 1,
   "request": {
     "method": "GET",
-    "url": "/GP0001/STU3/1/gpconnect/fhir/Patient?identifier=https://fhir.nhs.uk/Id/nhs-number%7CASDF"
+    "urlPattern": "\/.*\/STU3\/1\/gpconnect\/fhir\/Patient[?]identifier=https[:]\/\/fhir[.]nhs[.]uk\/Id\/nhs-number%7CASDF"
```

Have you tried double-backslashing? from [the documentation](http://wiremock.org/docs/request-matching/) it looks like some of this shouldn't be necessary
```suggestion
    "urlPattern": "/.*/STU3/1/gpconnect/fhir/Patient\\?identifier=https://fhir\\.nhs\\.uk/Id/nhs-number%7CASDF"
```